src/views/RoomSelector.vue

```diff
@@ -165,9 +167,10 @@ export default {
 
 <style lang="scss" scoped>
 #modal-inner {
-	width: 90vw;
-	max-width: 400px;
-	height: 55vh;
+	width: 100%;
+	height: 100%;
+	padding: 20px;
```

Comment:
After seing the screenshot I think we should dial this down to 12px

Reply:
@marcoambrosini 
It should be:
        height: 55vh;
	width: 100%;
	height: 100%;
	padding: 12px;
	
Correct? :)

Reply:
Correct!